cartridge/shop/models.py

```diff
@@ -129,6 +141,8 @@ def save(self, *args, **kwargs):
         if updating and not settings.SHOP_USE_VARIATIONS:
             default = self.variations.get(default=True)
             self.copy_price_fields_to(default)
+        elif hasattr(self, 'content_model'):
+            self.content_model = self._meta.object_name.lower()
```

Comment:
> It's a more consistent use of the field: if we leave this None for default Products, the field accurately represents the subclass for each instance.

I don't see this as inconsistent. The base class is the default content_model.

> If we don't, it represents the subclass or if there is none then the base class, which we then have to unpack every time we need to make that distinction.

Conversely, if we made the base class's content_model `None` it represents the content model or if it's the base class `None`, which we then have to unpack every time we *don't* want to make that distinction.

At any rate, I think it would have to be done in mezzanine's ContentModel which would need to handle backwards compatibility since it's already been pushed in 4.2.0. That's not a big deal, but I don't see how it's any easier to make this change now than it would be later, so why not wait until a use case for making the distinction arises?